app/helpers/sprints_helper.rb

```diff
@@ -1,2 +1,43 @@
 module SprintsHelper
+
+  def validate_sprint_date_parameter
+    errors = []
+
+    errors.append(
+        valid_date_range(params[:start_date] || '', params[:end_date] || '')
+    )
+    errors.append(
+        begins_in_the_future(params[:start_date] || '')
+    )
+
+    raise_exception_on_validation_error(errors)
+  end
+
+  private
+
+  def valid_date_range(start_date, end_date)
+    if end_date.empty? || start_date.empty?
+      return 'Start date and end date cannot be empty'
+    end
+
+    if end_date < start_date
+      'End date cannot be before the start date'
+    end
+  end
+
+  def begins_in_the_future(start_date)
+    if !start_date.empty?
+      if Date.parse(start_date) < Date.today
+        'Start date cannot be in the past'
```

Comment:
I'd change this so the END date can't be in the past - I can already see the "but what if I forgot to input this and want to start it retrospectively" coming... So as a general definition a sprint is only in the past if the END date is in the past.